lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -214,13 +260,7 @@ public Response resetMappings() {
         }
 
         try {
-            for (File mappingFile : mappings) {
-                if (mappingFile.exists()) {
-                    if (!mappingFile.delete()) {
-                        LOG.warn("Unable to delete mapping file " + mappingFile.toString());
-                    }
-                }
-            }
+            AtlasUtil.deleteDirectoryContents(mappingFolderPath.toFile());
```

Comment:
what about using Files.deleteIfExists(Path) or Files.delete(Path)?

Reply:
I'll look at Files.delete()

Reply:
decided to keep my AtlasUtil.deleteDirectoryContents() as it keeps the parent but removes all subcontent